src/resources/views/country/index.blade.php

```diff
@@ -0,0 +1,6 @@
+@extends('layouts.app')
+
+@section('content')
+    Country is {{ $response }}
```

Comment:
1. Sukurka failą `src/resources/lang/en/country.php` - tai bus vertimų failas
```
<?php

return [
    'country_affected_' => 'Country is not affected',
    'country_affected_1' => 'Country is affected',
];
```
2. Kreipkis į lang failą ir atspausdink vertimą:
```suggestion
    @lang('country.country_affected_' . $isAffected)
```
**DISCLAIMER:** Blade `bool` kintamuosius konvertuoja į skaičių `true = 1` arba `false = 0`. Dėl kažkokios "magijos" `false` gaunamas ne `0`, o tiesiog tuštuma. Dėl to kalbų faile reikia aprašyti ne `country_affected_0`, o `country_affected_`.

Turėk omenyje, kad rekomenduojama visus tekstus dėti į kalbų failus, net jei naudosi tik vieną kalbą. Jei ateityje sugalvotum išversti savo puslapį, tau pakaks išversti kalbų failus ir nereikės keisti visų blade turinio.